packages/aragon-wrapper/src/rpc/handlers/index.js

```diff
@@ -59,3 +59,5 @@ export { default as network } from './network'
 export { default as notifications } from './notifications'
 export { default as signMessage } from './sign-message'
 export { default as web3Eth } from './web3-eth'
+export { default as newAction } from './new-action'
```

Comment:
I think having a newAction and updateAction that do _exactly_ the same thing probably isn't great. I would either:
1. condense them into a single API endpoint (maybe like 'registerAction' or 'logAction') 
2. have the newAction at the _API_ level re-trigger the updateAction function with the correct state so the wrapper/messenger only handle one function.

I also think potentially the language of just having them be 'actions' may be too general.